src/Sushi.php

```diff
@@ -88,4 +89,9 @@ public function migrate()
 
         static::insert($rows);
     }
+
+    protected function getRows()
```

Comment:
This would mean that everyone has to define it even if the simple property approach would work. :/

Reply:
> This would mean that everyone has to define it even if the simple property approach would work. :/

Yeah i suppose, remember the trait is asking throws an error anyway if the ```$rows``` property is not not set, so if you make it abstract some IDEs will helpto remind you to override that function